python2/typing.py

```diff
@@ -1784,6 +1785,17 @@ def __new__(cls, *args, **kwds):
         return _generic_new(collections.defaultdict, cls, *args, **kwds)
 
 
+class Counter(collections.Counter, Dict[T, int]):
+    __slots__ = ()
+    __extra__ = collections.Counter
+
+    def __new__(cls, *args, **kwds):
+        if _geqv(cls, Counter):
+            raise TypeError("Type Counter cannot be instantiated; "
+                            "use Counter() instead")
```

Comment:
I would use full name ``collections.Counter()`` in the second line, otherwise this error message reads unclear.
(The same applies to Python 3 version).